test-status/api.js

```diff
@@ -185,7 +185,7 @@ function createErroredCheckParams(
         'error, please try the following steps:\n' +
         `1. Restart the failed [CI job](${ciJobUrl})\n` +
         '2. Rebase your pull request on the latest `master` branch\n' +
-        `3. Contact the weekly build on-duty (@${BUILD_ON_DUTY_TEAM}), who can advise you ` +
+        `3. Contact the weekly build cop (@${BUILD_COP_TEAM}), who can advise you ` +
```

Comment:
Is this being changed back to "build cop" intentionally, or is this a merge/rebase artifact?

Reply:
Good catch! It was a rebase conflict that I've fixed. (Checked for other such conflicts and found none.)